### PR TITLE
feat: IOS - ability to input sound volume

### PR DESCRIPTION
**Pull request by trowaflo**
Hi, 

I want to be able to manage the sound volume of critical notifications on iOS.
I want to be able to select the 'none' sound option and really have no sound, cause now none is like default.

Examples:
Have no sound, but still receive critical alerts, in silent mode.
Set the sound to 25% (because 100% in a silent context is quite annoying).

Please excuse in advance if the solutions I propose do not consider Android compatibility as I do not have any devices on which to test.

### CODE REVIEW

File: Frigate Camera Notifications/Stable [hunk 1]
```diff
@@ -193,6 +193,19 @@ blueprint:
             - default
             - none
           custom_value: true
+    volume:
+      name: Volume Sound - iOS only (Optional)
+      description: You can spécify a sound level between 0 and 100
+      default: "100"
+      selector:
+        select:
```
Comment (SgtBatten) [inline Frigate Camera Notifications/Stable:201]: Perhaps this could be a slider? 0 to 100

File: Frigate Camera Notifications/Stable [hunk 2]
```diff
@@ -648,53 +663,54 @@ action:
                     data_template:
                       name: Frigate Notification
                       message: |
-                        DEBUG:
```
Comment (SgtBatten) [inline Frigate Camera Notifications/Stable:651]: In the debug section the trailing whitespace is needed.

File: Frigate Camera Notifications/Stable [hunk 3]
```diff
@@ -730,8 +746,10 @@ action:
                               attachment:
                                 url: "/api/frigate/notifications/{{id}}/{{attachment}}.jpg"
                               push:
-                                sound: "{{sound}}"
-                                interruption-level: "{{ iif(critical, 'critical', 'active') }}"
+                                sound:
```
Comment (SgtBatten) [inline Frigate Camera Notifications/Stable:749]: This looks good according to quick review of the new companion docs. thanks

File: Frigate Camera Notifications/Stable [hunk 4]
```diff
@@ -730,8 +746,10 @@ action:
...
+                                  name: "{{sound}}"
+                                  volume: "{{ iif(sound == 'none', 0, volume) }}"
```
Comment (SgtBatten) [inline Frigate Camera Notifications/Stable:751]: How do we handle the default sound if people just want that. potentially causing issues for people who just want to enter the camera name and device and get notifications

**Comment by SgtBatten** [comment]
Pretty good. Thank you. 

I don't believe this would introduce any issues for android. 

I don't want to add anything directly to stable. Please add the necessary changes to the beta instead after considering my comments above.

File: Frigate Camera Notifications/Stable (hunk 4 above)
Comment (trowaflo) [reply Frigate Camera Notifications/Stable:751]: "default" sound is already `default: default` and "default" volume is `default: 100`

So people who specify neither of the two have 100% volume of default sound, like now.

--
When "none" is selected, the sound is set to "0".

However, if nothing is selected, the volume is taken as volume: "{{ iif(sound == 'none', 0, volume) }}".
 
If the volume is '0', there will be no sound regardless of the selected option for the sound, it's like sound is 'none'.

File: Frigate Camera Notifications/Stable (hunk 1 above)
Comment (trowaflo) [reply Frigate Camera Notifications/Stable:201]: Absolutely, very good idea.

**Comment by trowaflo** [comment]
Thank to you for your reviews. I will move to beta once all issues have been resolved.

File: Frigate Camera Notifications/Stable (hunk 2 above)
Comment (SgtBatten) [reply Frigate Camera Notifications/Stable:651]: Necessary perhaps is a bit of a stretch. It's just formatting. 

If people view the debug messages via the trace it's perfectly fine. 

If people view it via the logbook the debug message doesn't respect the new lines so the spaces just make it more legible. That's all.

**Comment by SgtBatten** [comment]
Looks good for transposing into the beta. Thanks

**Comment by trowaflo** [comment]
Transpose to beta is done. 

For my information, what is the lifecycle time for the Beta? And just to understand, why use Beta instead of a development or feature branch? It seems easier to track all features that way.

**Comment by SgtBatten** [comment]
Okay i think this is good to go, the only thing is this pull request still has the stable branch changes. 

If you can copy the raw github file without using your text editor and paste it into a new clean pull request I'll merge it.

**Comment by trowaflo** [comment]
It's strange, I rolled back the stable file. 
I also disabled whitespace trimming and update beta file.
For me, the file change view is okay. Is it okay for you too?

**Pull request merged**